package.json

```diff
@@ -18,14 +19,19 @@
     "@types/node": "18.11.18",
```

Comment:
I dunno why vercel team have put these `@types/...` dependencies into production deps - try to move them into devDependencies, then build to see if all ok.